sphinx_panels/__init__.py

```diff
@@ -70,8 +166,32 @@ def add_static_path(app):
         app.config.html_static_path.append(static_path)
 
 
+def visit_container(self, node):
+    classes = "docutils container"
+    if node.get("in_panel", False):
+        # we don't want the CSS for container for these nodes
+        classes = "docutils"
+    self.body.append(self.starttag(node, "div", CLASS=classes))
+
+
+def depart_container(self, node):
+    self.body.append("</div>\n")
+
+
 def setup(app):
     app.add_directive("panels", Panels)
     app.connect("builder-inited", add_static_path)
+
     app.add_css_file("panels.css")
+
+    # TODO only load these is using a non-boostrap theme
+    app.add_css_file("bs-grids.css")
+    app.add_css_file("bs-cards.css")
```

Comment:
Here I'd like to find a way to only load these CSS if `bootstrap.min.css` is not already being loaded (they just contain copied CSS from that, specific to each element)

Reply:
I can't see any easy way to automate bootstrap theme discovery so, in e67f3ac, I have added a sphinx option to disable loading the bootstrap CSS